README.md

```diff
@@ -14,3 +14,10 @@ Your statements should include:
     * **Including an fake data set**
 
 Good luck!
+
+--
+I would like to change my problem design statement entirely. 
+Aside from the theatre, I am passionate about music and technology. Particularly, I am a big fan of music streaming. I genuinely believe that music streaming is the future of the industry, and it has made music more accessible to people. However, there are many critics who believe that music streaming is hurting the industry, and that music streaming services do not pay artists enough money.
+
+I want to better understand the impact of music streaming services on the music industry based on revenues and identify which category of artists benefit from or are hurt by streaming services. In completing this goal, I hope to clarify the misconceptions surrounding music streaming services, and if necessary, inform better royalty pool structures. For this project, I plan to collect data on revenues earned in the music industry in the US by format (CD sales, concert ticket sales, vinyl sales, etc.) over 40+ years. Then, I will categorize artists based on their popularity by looking at social media likes/follows, and will collect data on their estimated revenues based on the existing royalty pool structures established by streaming services. Because my main goal is to demistify the impact music streaming services have on the music industry, my priority will lie in collecting data on revenues earned in the music industry.
```

Comment:
This project will hinge on whether you can get royalty info for individual artists and can find a trend. Is this kind of revenue data available?